src/StockportContentApi/ContentfulFactories/CallToActionBannerContentfulFactory.cs

```diff
@@ -7,6 +7,9 @@ public class CallToActionBannerContentfulFactory : IContentfulFactory<Contentful
     {
         public CallToActionBanner ToModel(ContentfulCallToActionBanner entry)
         {
+            if (entry is null)
+                return null;
+
```

Comment:
is this needed?

Reply:
I moved the previous null check from src/StockportContentApi/ContentfulFactories/CommsContentfulFactory.cs  line: 26